content/en/docs/reference/command-line-tools-reference/feature-gates.md

```diff
@@ -702,6 +703,7 @@ Each feature gate is designed for enabling/disabling a specific feature:
   for all workloads.
   The seccomp profile is specified in the `securityContext` of a Pod and/or a Container.
 - `SecurityContextDeny`: This gate signals that the `SecurityContextDeny` admission controller is deprecated.
+- `SeparateTaintEvictionController`: Enables running `TaintEvictionController` - that performs [Taint Based Evictions](/docs/concepts/scheduling-eviction/taint-and-toleration/#taint-based-evictions) - in a separated controller from `NodeLifecycleController`.
```

Comment:
Please wrap the long lines manually.
Maybe we can add a sentence or two articulating the user perceivable impacts from this.
As a user I need to know whether/when to turn this gate off.
```suggestion
- `SeparateTaintEvictionController`: Enables running `TaintEvictionController`
  that performs [Taint Based Evictions](/docs/concepts/scheduling-eviction/taint-and-toleration/#taint-based-evictions)
  in a separated controller from `NodeLifecycleController`.
```